sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/README.md

```diff
@@ -1,34 +1,66 @@
 # Azure Monitor Live Metrics client library for .NET
 
-TODO
+The [OpenTelemetry .NET](https://github.com/open-telemetry/opentelemetry-dotnet) compatible processor which sends [telemetry data](https://docs.microsoft.com/azure/azure-monitor/app/data-model) to [Azure Monitor](https://docs.microsoft.com/azure/azure-monitor/app/app-insights-overview) [Live Metrics](https://learn.microsoft.com/azure/azure-monitor/app/live-stream).
 
 ## Getting started
 
+### Prerequisites
+
+- **Azure Subscription:**  To use Azure services, including Azure Monitor Exporter for [OpenTelemetry .NET](https://github.com/open-telemetry/opentelemetry-dotnet), you'll need a subscription.  If you do not have an existing Azure account, you may sign up for a [free trial](https://azure.microsoft.com/free/dotnet/) or use your [Visual Studio Subscription](https://visualstudio.microsoft.com/subscriptions/) benefits when you [create an account](https://azure.microsoft.com/account).
+- **Azure Application Insights Connection String:** To send telemetry data to the monitoring service you'll need connection string from Azure Application Insights. If you are not familiar with creating Azure resources, you may wish to follow the step-by-step guide for [Create an Application Insights resource](https://docs.microsoft.com/azure/azure-monitor/app/create-new-resource) and [copy the connection string](https://docs.microsoft.com/azure/azure-monitor/app/sdk-connection-string?tabs=net#find-your-connection-string).
+
 ### Install the package
 
+Install the Azure Monitor Live Metrics client for OpenTelemetry .NET with [NuGet](https://www.nuget.org/):
+```dotnetcli
+dotnet add package Azure.Monitor.OpenTelemetry.LiveMetrics --prerelease
+```
 
-### Prerequisites
+#### Nightly builds
+
+Nightly builds are available from this repo's [dev feed](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#nuget-package-dev-feed).
+These are provided without support and are not intended for production workloads.
+
+### Add the Live Metrics processor
 
+The following examples demonstrate how to add the `LiveMetricsExtractionProcessor` to your OpenTelemetry configuration.
```

Comment:
`LiveMetricsExtractionProcessor` is an internal detail. Call it Live metrics exporter?

Reply:
let me change this to "Live Metrics client", which I used everywhere else in the document